service/src/main/java/bio/terra/cbas/dependencies/wds/WdsClient.java

```diff
@@ -46,22 +37,15 @@ protected ApiClient getApiClient()
 
     ApiClient apiClient = new ApiClient().setBasePath(uri);
     apiClient.setHttpClient(singletonHttpClient);
-    apiClient.addDefaultHeader(
-        "Authorization",
-        "Bearer " + credentialLoader.getCredential(CredentialLoader.CredentialType.AZURE_TOKEN));
+    apiClient.addDefaultHeader("Authorization", "Bearer " + accessToken);
```

Comment:
Do you know if there's any meaningful difference between this and `setAccessToken`?